COUNTER_Robots_list.json

```diff
@@ -225,6 +229,10 @@
     "last_changed": "2020-02-25",
     "url": "https://www.mediawiki.org/wiki/Citoid"
   },
+  {
+    "pattern": "ClaudeBot",
```

Comment:
This would be matched by the generic `bot` pattern (and remember, this list is supposed to be use with case insensitivity).